wordpress-mobile/gutenberg-mobile
# Bump Jest version to v27

**Related to https://github.com/WordPress/gutenberg/pull/38133.**

This PR bumps the Jest package version to v27 to match the same version in Gutenberg, which was recently updated in https://github.com/WordPress/gutenberg/pull/33287.

Additionally, it updates the unit tests located in GB-mobile in order to prevent failures with the update.

To test:

### Automated tests
1. Check that all PR checks pass, especially the Android, iOS device tests, and unit tests.

### Manual tests
1. Run command `npm run test:e2e:android:local`.
2. Observe that Android device tests pass.
3. Run command `npm run test:e2e:ios:local`.
4. Observe that iOS device tests pass. 
5. Run command `npm run test`.
6. Observe that unit tests pass. 

PR submission checklist:
